sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -36,7 +36,7 @@ void IJsonModel<AudioTranscriptionOptions>.Write(Utf8JsonWriter writer, ModelRea
             }
             if (ResponseFormat.HasValue)
             {
-                writer.WritePropertyName("response_format"u8);
+                writer.WritePropertyName("responseFormat"u8);
```

Comment:
All those property Name changes is because openai has changed to use `@encodeName` which is not supported in .net emitter. 
we need to commit this and then upgrade typespec 0.53 and then support @encodeName